manage_user_delete.php

```diff
@@ -62,10 +62,11 @@
 # current user.
 access_ensure_global_level( $t_user['access_level'] );
 
-# check that we are not deleting the last administrator account
+# Check that we are not deleting the last administrator account
 $t_admin_threshold = config_get_global( 'admin_site_threshold' );
-if( user_is_administrator( $f_user_id ) &&
-	 user_count_level( $t_admin_threshold ) <= 1 ) {
+if( (int)$t_user['enabled'] === 1 &&
```

Comment:
Good point @atrol.  I've opened a dedicated bug for this and fixed it as part of this pull request.